components/accordion-menu/template.html

```diff
@@ -0,0 +1,3 @@
+<component-slot data-name="accordion-panel">
+
+</component-slot>
```

Comment:
The slots are useful in the cases where you need to put something in a specific place.
In this case the initial html in the demo is the same as the one after the render.
We can safely remove the render method and leave the connectedCallback like this:

    connectedCallback() {
        //Check if it has the multiple attribute
        this._multiple = this.hasAttribute('multiple');
        //Remove the border from the last panel
        this.lastChild.classList.add('accordion-last-panel');

        this.addEventListener('click', this.panelClick);
    }
    
    Refer to the slots usage guide for more details - https://developer.mozilla.org/en-US/docs/Web/Web_Components/Using_templates_and_slots

https://slack-imgs.com/?c=1&o1=ro&url=https%3A%2F%2Fuser-images.githubusercontent.com%2F19714551%2F133610228-8544b886-fe22-4799-b471-841ed3053427.png